<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>InvokeMethod</class>
  <widget class="QDialog" name="InvokeMethod" >
    <property name="objectName" >
      <string notr="true" >InvokeMethod</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>503</width>
        <height>416</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Invoke Methods</string>
    </property>
    <layout class="QGridLayout" >
      <property name="objectName" >
        <string notr="true" >unnamed</string>
      </property>
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item rowspan="1" row="1" column="0" colspan="2" >
        <widget class="QGroupBox" name="boxParameters" >
          <property name="objectName" >
            <string notr="true" >boxParameters</string>
          </property>
          <property name="title" >
            <string>Parameter &amp;List</string>
          </property>
          <layout class="QGridLayout" >
            <property name="objectName" >
              <string notr="true" >unnamed</string>
            </property>
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item rowspan="1" row="0" column="0" colspan="3" >
              <widget class="QTreeWidget" name="listParameters" >
                <property name="objectName" >
                  <string notr="true" >listParameters</string>
                </property>
                <column>
                  <property name="text" >
                    <string>Parameter</string>
                  </property>
                  <property name="clickable" >
                    <bool>false</bool>
                  </property>
                  <property name="resizeable" >
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text" >
                    <string>Type</string>
                  </property>
                  <property name="clickable" >
                    <bool>false</bool>
                  </property>
                  <property name="resizeable" >
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text" >
                    <string>Value</string>
                  </property>
                  <property name="clickable" >
                    <bool>false</bool>
                  </property>
                  <property name="resizeable" >
                    <bool>true</bool>
                  </property>
                </column>
              </widget>
            </item>
            <item row="1" column="2" >
              <widget class="QToolButton" name="buttonSet" >
                <property name="objectName" >
                  <string notr="true" >buttonSet</string>
                </property>
                <property name="text" >
                  <string>&amp;Set</string>
                </property>
              </widget>
            </item>
            <item row="1" column="1" >
              <widget class="QLineEdit" name="editValue" >
                <property name="objectName" >
                  <string notr="true" >editValue</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QLabel" name="TextLabel3" >
                <property name="objectName" >
                  <string notr="true" >TextLabel3</string>
                </property>
                <property name="text" >
                  <string>Parameter &amp;Value:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>editValue</cstring>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="1" row="0" column="0" colspan="2" >
        <layout class="QGridLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item row="1" column="1" >
            <widget class="QLineEdit" name="editReturn" >
              <property name="objectName" >
                <string notr="true" >editReturn</string>
              </property>
              <property name="readOnly" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1" >
            <widget class="QComboBox" name="comboMethods" >
              <property name="objectName" >
                <string notr="true" >comboMethods</string>
              </property>
              <property name="editable" >
                <bool>true</bool>
              </property>
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="autoCompletion" >
                <bool>true</bool>
              </property>
              <property name="duplicatesEnabled" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="0" >
            <widget class="QLabel" name="labelMethods" >
              <property name="objectName" >
                <string notr="true" >labelMethods</string>
              </property>
              <property name="text" >
                <string>&amp;Method Name:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>comboMethods</cstring>
              </property>
            </widget>
          </item>
          <item row="0" column="2" >
            <spacer name="Spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Preferred</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item row="1" column="0" >
            <widget class="QLabel" name="TextLabel1" >
              <property name="objectName" >
                <string notr="true" >TextLabel1</string>
              </property>
              <property name="text" >
                <string>Returned Value:</string>
              </property>
            </widget>
          </item>
          <item row="0" column="3" >
            <widget class="QPushButton" name="buttonInvoke" >
              <property name="objectName" >
                <string notr="true" >buttonInvoke</string>
              </property>
              <property name="text" >
                <string>In&amp;voke</string>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item rowspan="1" row="1" column="2" colspan="2" >
            <spacer name="Spacer5" >
              <property name="sizeHint" >
                <size>
                  <width>111</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item row="2" column="0" >
        <spacer name="Spacer4" >
          <property name="sizeHint" >
            <size>
              <width>361</width>
              <height>21</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="2" column="1" >
        <widget class="QPushButton" name="PushButton2" >
          <property name="objectName" >
            <string notr="true" >PushButton2</string>
          </property>
          <property name="text" >
            <string>Close</string>
          </property>
          <property name="autoDefault" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
